src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py

```diff
@@ -21,13 +21,13 @@ def __init__(self, input_h, input_w, batch_size, output_size, out_h, out_w, init
         self.gpu_policy = _utils.TensorFlowGPUPolicy()
         self.gpu_policy.start()
 
-        #reset tensorflow graph when a new model is created
-        _tf.reset_default_graph()
 
         # Converting incoming weights from shared_float_array to numpy
         for key in init_weights.keys():
             init_weights[key] = _utils.convert_shared_float_array_to_numpy(init_weights[key])
+        self.init_weights = init_weights
```

Comment:
Could we instead just pass `init_weights` as an argument to `self.init_object_detector_graph`? Doesn't look like we need it afterwards.

Reply:
Thats a good idea! Done